frontend/www/js/omegaup/components/course/Statistics.vue

```diff
@@ -295,6 +295,7 @@ export default class Statistics extends Vue {
     return {
       chart: {
         type: 'bar',
+        height: `${data.length < 10 ? null : (data.length * 10) / 2}%`,
```

Comment:
se puede hacer que esto no despliegue `null%`?

```suggestion
        height: data.length < 10 ? null : `${data.length * 5}%`,
```